src/App.js

```diff
@@ -20,13 +21,18 @@ store.subscribe(function(){console.log(store.getState())});
 store.subscribe(saveState);
 
 class App extends Component {
-  render() {
-    return (
-    		<Provider store={store}>
-    			<UploadPageContainer/>
-    		</Provider>
-    );
-  }
+  
+	componentWillMount() {
```

Comment:
componentWillMount is deprecated in 16.3 I think you can just use componentDidMount here?

Reply:
Can we address this as technical debt (I know we are piling up a bunch right now)?

Reply:
It's your call. I would just switch it and if it works it works but if it doesn't flip it back